core/disputes.go

```diff
@@ -176,6 +172,8 @@ func (n *OpenBazaarNode) verifyEscrowFundsAreDisputeable(contract *pb.RicardianC
 			log.Errorf("Failed GetConfirmations(%s): %s", hash.String(), err.Error())
 			return false
 		}
+		confirmationsForTimeout := contract.VendorListings[0].Metadata.EscrowTimeoutHours *
+			uint32(repo.ConfirmationsPerHour(order.Payment.AmountCurrency.Code))
```

Comment:
And since you're casting to `uint32` here and not using `ConfirmationsPerHour` anywhere else, why don't you just change the return type of the function?